### ISSUE TITLE
Black screen, unintelligible VI/s (Wario patch)

### ISSUE THREAD

**Issue by dahuterschuter** [JSsixtyfour/smashremix#14]
tl;dr - Patching the Smash 64 ROM with the latest Wario patch results in a smashremix0.9.2.z64 file that only creates a black screen when opened in Project64 2.3.2.

On the latest Wario build.  Windows 10.

I drag either:
>Super Smash Bros. (U) [!] .zip
or
>Super Smash Bro.s (USA) .zip
onto the windows-drag-here patcher in the smashremix0.9.2 folder.

Doing so results in the message:  "Your ROM does not match the developer's original ROM." and a failure to write after going through some cmd processes.

If I instead drag the Super Smash Bros. (U) [!] .z64 file onto the windows-drag-here patcher, it seems to write, and gave me a file in the output.

When I attempt to open that outputted file, it only results in a black screen with VI/s numbers all over the place.  

The (USA) .z64 ROM outputs something playable to the ROM's .zip file simply called input.z64, but this ROM has no changes and is simply a Smash 64 game when opened with no additional characters.

Any idea what is going wrong?  I'm using Project64 2.1.  The directions are simple enough to follow, yet yielding failures, so I'm not sure what could be the problem.

EDIT:  Updated to Project64 2.3.2 to check if that was the issue, however this still only results in a black screen, but the VI/s stabilizes at 60/61.  Have tried this on two different machines with three different ROMs.

**Comment by jordanbarkley** [comment]
.zip files are not ROM files. Anything can be inside a .zip file. 

You want a LEGALLY ACQUIRED file (i.e. one you got yourself using a cartridge and and external tool). These typically have a .z64 extension. 

You then need to use drag-rom-here patcher as described in your response and grab that output file. 

Next, make sure you enable the expansion pak in Projext64. Open the output file as you described. After you get the black screen, go to Options (at the top) >> Settings >> Config: SMASH REMIX >> Memory Size and change that option to 8 MB. Press APPLY. Then just reopen the ROM.
